### PR TITLE
Package shared framework symbols to a separate compression file

### PR DESCRIPTION
**Pull request by chcosta**
Produces  separate zip / tarballs with symbols included.  Naming convention is 

dotnet-sharedframework-[os]-[version].[extension] (product) => dotnet-sharedframework-symbols-[os]-[version].[extension] (symbols)

Also updating the version of buildtools in this repo to include the fix which allows building in a vs2017 dev command prompt.

/cc @ellismg @weshaggard @Petermarcu

### CODE REVIEW

**Comment by ellismg** [comment]
You probably do not want to delete the test targets file.

**Comment by chcosta** [comment]
yikes, thanks.  I consistently do that to speed up my dev loop.  i'll add it back.

**Comment by ellismg** [comment]
> yikes, thanks. I consistently do that to speed up my dev loop. i'll add it back.

In the future, in my dev loop I pass --targets Init,Compile,Package (I think on windows it's -Targets instead) to skip the testing targets but do the build and packaging.

**Comment by chcosta** [comment]
It's not clear to me what the buildtools failure is.  i'm decoupling that change from this update.  i'll investigate the problem in corefx and open a new PR in core-setup when it's understood.

File: build_projects/shared-build-targets-utils/Utils/SharedFrameworkPublisher.cs [hunk 1]
```diff
@@ -160,6 +160,14 @@ public void PublishSharedFramework(string outputRootDirectory, string commitHash
             var content = $@"{commitHash}{Environment.NewLine}{version}{Environment.NewLine}";
             File.WriteAllText(Path.Combine(sharedFrameworkNameAndVersionRoot, ".version"), content);
 
+            // Populate symbols publish folder
+            string sharedFrameworkNameAndVersionWithSymbolsRoot = $"{outputRootDirectory}.symbols";
```
Comment (weshaggard) [inline build_projects/shared-build-targets-utils/Utils/SharedFrameworkPublisher.cs]: It is kind of annoying to see the duplication of the path creation here... but I will leave that along until we covert to BuildTools.

**Comment by Petermarcu** [comment]
Thanks @chcosta , I assume you want to add links to these new archives to the readme when this is all working?

**Comment by chcosta** [comment]
Yes, I created the separate tracking issue referenced above because I want to validate the blobs are present before linking them - https://github.com/dotnet/core-setup/issues/1756

**Comment by Petermarcu** [comment]
Perfect!

**Pull request merged**